### ISSUE TITLE
New Release?

### ISSUE THREAD

**Issue by bitti** [ledger/ledger-mode#330]
Seems the last release tag approaches 3 years. This means fixes like https://github.com/ledger/ledger-mode/commit/bcd8cefb720702db88986a52bb66e08e2e451c05 are not in melpa-stable which in turn means `ledger-mode` is broken in newer emacsen.

Can we make a new release?

**Comment by g-gundam** [comment]
Although it hasn't been tagged, the most recent release on MELPA of ledger-mode has commits from Nov 14, 2024.  This issue should be closed.

https://github.com/ledger/ledger-mode/tree/15b7d29f2539f9e9671ab3c062bd5165e5b80ae8

![Image](https://github.com/user-attachments/assets/7d6e72ad-be6a-4cac-8920-8788d5a38fc5)

**Comment by purcell** [comment]
> the most recent release on MELPA of ledger-mode

Yes, but these are explicitly snapshot packages, so the request for a fresh stable release is still broadly reasonable — only such released versions would find their way into Debian and other software distributions.

**Comment by Thaodan** [comment]
I'm packaging ledger-mode for openSUSE. I had the same concern notably when the current tagged release might not work against the last Emacs version released.

**Comment by bcc32** [comment]
Unfortunately due to work I haven't had any time to dedicate to this project in the last couple months, but I'm hoping to take a look at a couple live issues/PRs in the next week or two that would be great to include in the next release.  Would also be happy to help with the release process!

**Comment by pabl0** [comment]
I think even just tagging the current master as a new release would be an improvement? Even the NEWS.org file have been kept up-to-date pretty well. Is 4.0.0 really much more "stable" right now?

**Comment by viiru-** [comment]
Commit d31c22dd9ebbdfd31a4fe492b872388232685898 by @jwiegley bumped the version number up to 4.1.0. Could we perhaps get a tag pointing at that commit?
